src/test/java/com/stripe/functional/AccountTest.java

```diff
@@ -95,6 +95,29 @@ public void testUpdate() throws StripeException {
     );
   }
 
+  @Test
+  public void testUpdateDocument() throws StripeException {
```

Comment:
Not sure this test is worth it -- it basically only tests that stripe-mock accepts the new parameter. I'd probably just remove it.